### DISCUSSION TITLE
Welcome to pulsar-client-dotnet Discussions!

### DISCUSSION THREAD

**Discussion by Lanayx** [fsharplang-ru/pulsar-client-dotnet#130] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by franck-schmidlin** [comment]
Hi there.

At this stage I am only just starting with Streaming in general and Pulsar in particular. I have written some PoC code a while back to test the concept, and since then just keep an eye on it.
Having tried all 3  dot net pulsar client libraries, pulsar-client-dotnet has proven the best so far for my limited uses case.

Thanks and keep up the good work!
